### DISCUSSION TITLE
About WGC yellow border

### DISCUSSION THREAD

**Discussion by zyxeeker** [WindowTop/WindowTop-App#373] [Q&A] [answered]
Hi, I saw ur past [issue](https://github.com/WindowTop/WindowTop-App/issues/229) and the [reply](https://github.com/microsoft/Windows.UI.Composition-Win32-Samples/issues/97) about WGC yellow border.
I want to known how to sloved it in Win10 older version? Thanks!

**Comment by gileli121** [comment, marked as answer]
Hello, this is not possible on older Windows 10 versions because the OS API to disable this border is unavailable.
The only option is to update your Windows 10 version to the most recent version.
What Windows 10 version do you have? You can check it in the winver app (press start and type "winver").
Send me a screenshot.

There may be a bug that causes WindowTop to think that it is not supported anyway (because you use Windows 10), so it does not utilize the new feature in the OS API.

**Reply by zyxeeker** [reply]
Thanks! I use Windows 11 to develop the sdk but my users not use newer Windows. So I try it in Windows 10 19045 and I found that. 
I will try other way to do
Thanks again!
